src/consts/consts.js

```diff
@@ -0,0 +1,49 @@
+const blankStar = { x: 0, y: 0, radius: -1, fill: "black", name: "" };
+
+const fillArray = [

```

Comment:
I don't see this coming back anywhere in the code, am I missing something?

Reply:
this is used in App to get a random color for stars.  I renamed this to starColorArray to make it clearer. I think there's probably a much better way to do this than how I'm doing it.